lib/sidekiq/statsd/server_middleware.rb

```diff
@@ -1,7 +1,5 @@
 # encoding: utf-8
 
-require 'statsd'
```

Comment:
@kevwilde This would fail with `dogstatsd-ruby` - I moved it to `initialize_statsd`.